### PR TITLE
UI2 - Create wallet choice scene

### PR DESCRIPTION
**Pull request by peachbits**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202155384662000

### CODE REVIEW

**Comment by peachbits** [comment]
Simulator Screen Shot - iPod touch (7th generation) - 2022-04-20 at 15 03 59

File: src/components/scenes/CreateWalletChoiceScene.js [hunk 1]
```diff
@@ -22,90 +19,57 @@ type OwnProps = {
...
+  const onSelectNew = () => {
     navigation.navigate('createWalletSelectFiat', {
       selectedWalletType
     })
   }
 
-  onSelectRestore = () => {
-    const { navigation, route } = this.props
-    const { selectedWalletType } = route.params
+  const onSelectRestore = () => {
```
Comment (paullinator) [inline src/components/scenes/CreateWalletChoiceScene.js:34]: use `useCallback`

File: src/components/scenes/CreateWalletChoiceScene.js [hunk 2]
```diff
@@ -22,90 +19,57 @@ type OwnProps = {
...
-  onSelectNew = () => {
-    const { navigation, route } = this.props
-    const { selectedWalletType } = route.params
+export const CreateWalletChoiceScene = (props: Props) => {
+  const { navigation, route } = props
+  const { selectedWalletType } = route.params
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  const onSelectNew = () => {
```
Comment (paullinator) [inline src/components/scenes/CreateWalletChoiceScene.js:28]: use `useCallbackà

File: src/components/scenes/CreateWalletChoiceScene.js [hunk 3]
```diff
@@ -22,90 +19,57 @@ type OwnProps = {
...
-                </SecondaryButton>
-              </View>
-            </View>
-          </ScrollView>
-        </View>
-      </SafeAreaView>
-    )
-  }
+  // Should create svg scaler util
+  const svgScale = theme.rem(1 / 16)
```
Comment (paullinator) [inline src/components/scenes/CreateWalletChoiceScene.js:41]: why all the magic number?

File: src/components/scenes/CreateWalletChoiceScene.js [hunk 4]
```diff
@@ -22,90 +19,57 @@ type OwnProps = {
...
-    width: scale(64)
-  },
-  createWalletPromptArea: {
-    paddingTop: scale(16),
-    paddingBottom: scale(8)
+const getStyles = cacheStyles((theme: Theme) => ({
+  icon: {
+    flexDirection: 'row',
+    justifyContent: 'center',
+    marginTop: theme.rem(1),
```
Comment (paullinator) [inline src/components/scenes/CreateWalletChoiceScene.js:64]: Can just do `marginVertical`

File: src/components/scenes/CreateWalletChoiceScene.js [hunk 5]
```diff
@@ -22,90 +19,57 @@ type OwnProps = {
...
+  return (
+    <SceneWrapper avoidKeyboard background="theme">
+      <SceneHeader withTopMargin title={s.strings.title_create_wallet} />
+      <View style={styles.icon}>
+        <CreateWalletSvg color={theme.iconTappable} width={svgWidth} height={svgHeight} />
+      </View>
+      <EdgeText style={styles.instructionalText} numberOfLines={2}>
+        {s.strings.create_wallet_choice_instructions}
+      </EdgeText>
+      <MainButton alignSelf="stretch" label={s.strings.create_wallet_choice_new_button} marginRem={[1, 1]} type="secondary" onPress={onSelectNew} />
```
Comment (paullinator) [inline src/components/scenes/CreateWalletChoiceScene.js:54]: Declare the [1, 1] as a const outside of component.

**Review by paullinator — CHANGES_REQUESTED** [review]

File: src/components/scenes/CreateWalletChoiceScene.js [hunk 6]
```diff
@@ -22,90 +21,69 @@ type OwnProps = {
...
-  next: {
-    marginLeft: scale(1),
-    flex: 1
+    fontSize: theme.rem(1),
+    color: theme.primaryText,
+    paddingHorizontal: theme.rem(1),
+    marginTop: theme.rem(0.5),
+    marginBottom: theme.rem(2),
+    marginRight: theme.rem(2.5),
+    marginLeft: theme.rem(2.5),
```
Comment (thehobbit85) [inline src/components/scenes/CreateWalletChoiceScene.js:86]: marginHorizontal: theme.rem(2.5) 

;)

**Review by thehobbit85 — CHANGES_REQUESTED** [review]

**Review by thehobbit85 — APPROVED** [review]

**Pull request merged**